keps/sig-cli/3659-kubectl-apply-prune/README.md

```diff
@@ -627,128 +544,171 @@ In future, we may define additional “index” mechanisms here to further optim
 
 In pseudo-code, to discover the existing members of an ApplySet:
 
 ```bash
-for-each gk in $(split group-kind-annotation); do
-kubectl get $gk -n ns -l  applyset.k8s.io/id  # --only-partial-object-metadata
+for-each gk in $(split group-resources-annotation); do
+kubectl get $gk -n ns -l  applyset.kubernetes.io/id  # --only-partial-object-metadata
 done
 ```
 
-If the `applyset.k8s.io/additional-namespaces` annotation is present, any namespaced queries will need to be repeated for each target namespace.
+If the `applyset.kubernetes.io/additional-namespaces` annotation is present, any namespaced queries will need to be repeated for each target namespace.
 
-If the list in the `contains-group-kinds` annotation includes namespaced-scoped GKs on a cluster-scoped parent with no `applyset.k8s.io/additional-namespaces` annotation, kubectl will output an error.
+If the list in the `contains-group-resources` annotation includes namespaced-scoped GRs on a cluster-scoped parent with no `applyset.kubernetes.io/additional-namespaces` annotation, kubectl will output an error.
 
-If the contains-group-kinds annotation is missing, kubectl will initially consider this an error. Based on feedback, we can consider either falling back on a (very slow) full-GK scan to populate the annotation (after confirming kubectl owns the parent), or pointing users to a separate command (similar in spirit to `fsck`) that will do so. We will add warnings/suggestions to the main "apply" flow when we detect problems that might require a full-scan / discovery.  We may extend this based on user-feedback from the alpha.
+If the contains-group-resources annotation is missing, kubectl will initially consider this an error. Based on feedback, we can consider either falling back on a (very slow) full-GR scan to populate the annotation (after confirming kubectl owns the parent), or pointing users to a separate command (similar in spirit to `fsck`) that will do so. If we do the latter, we will add warnings/suggestions to the main "apply" flow when we detect problems that might require a full-scan / discovery.  We may extend this based on user-feedback from the alpha.
 
-Based on performance feedback, we can also consider switching to the alternative `applyset.k8s.io/inventory` hint annotation. Even if we do not trust the GKNN list for deletion purposes (we cannot, as it is not the source of truth), it could be used to optimize certain specific cases, most notably the no-op scenario where the current set exactly matches the list.
+Based on performance feedback, we can also consider switching to the alternative `applyset.kubernetes.io/inventory` hint annotation. Even if we do not trust the GKNN list for deletion purposes (we cannot, as it is not the source of truth), it could be used to optimize certain specific cases, most notably the no-op scenario where the current set exactly matches the list.
```

Comment:
Wdyt about adding testing perf and making a decision on whether to support this as a beta criterion?